account_asset_management/models/account_asset.py

```diff
@@ -749,8 +757,12 @@ def compute_depreciation_board(self):
                         line["amount"] = amount_diff
                     # compensate in first depreciation entry
                     # after last posting
-                    line = table[table_i_start]["lines"][line_i_start]
-                    line["amount"] -= amount_diff
+                    else:
+                        for entry in table[table_i_start:]:
+                            [
+                                line.update({"amount": line["amount"] - amount_diff})
```

Comment:
I'm reluctant to split the difference amongst all the remaining depreciations instead of putting the diff in the first available one. I will ask for the opinion of @luc-demeyer in this matter.

Reply:
ping @luc-demeyer